### PR TITLE
allow non-standard tape serial regex workaround

### PR DESCRIPTION
**Pull request by dericed**
resovles https://github.com/amiaopensource/ltopers/issues/167

### CODE REVIEW

**Comment by retokromer** [comment]
Why not provide the regex that matches the correct 8-char possibilities? In my opinion, this would be more useful IRL.

**Comment by dericed** [comment]
- IRL, i suspect it would be useful for either non-standard lengths or specifically to limit the regex to local implementations (constraints to letter usage or patterns), but ya the example could communicate that more clearly

**Comment by retokromer** [comment]
As long as I know, my original regex with the addition of the mentioned «bastard» would cover all possibilities in the wild today.

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ brew upgrade
 
 This will update Homebrew and upgrade every package you've installed using it.
 
+By default, ltoper scripts require that tape serials are 6 character strings of numbers and capital letters. Alternate regular expressions may be provided by setting LTOPER_TAPE_SERIAL_REGEX as a global variable. For instance: `export LTOPER_TAPE_SERIAL_REGEX="^[Z1-2]{8}$" ; ./formatlto` would permit an 8 character string comprised only of capital Z, number 1, and number 2 as a tape serial.
```
Comment (retokromer) [inline README.md]: typo: ltoper scripts

**Pull request merged**